src/components/admin/localization-statuses/LocalizationStatuses.tsx

```diff
@@ -0,0 +1,50 @@
+import {
+    EntityLocalization,
+    EntityWithLocalizations,
+    LocalizationLanguage,
+    TranslationStatus,
+} from '../../../types/common/language';
+import './LocalizationStatuses.scss';
+
+export interface LocalizationStatusProps<TLocalization extends EntityLocalization> {
+    languages: LocalizationLanguage[];
+    localizedEntity: EntityWithLocalizations<TLocalization>;
+}
+
+export const LocalizationStatuses = <TLocalization extends EntityLocalization>({
+    languages,
+    localizedEntity,
+}: LocalizationStatusProps<TLocalization>) => {
+    return (
+        <div className="statuses" data-testId="localization-statuses">
+            {languages.map((language) => (
+                <span
+                    key={language.id}
+                    className={`badge ${getLocalizationClassNameFromStatus(language, localizedEntity)}`}
+                >
+                    {language.code.toUpperCase()}
+                </span>
+            ))}
+        </div>
+    );
+};
+
+export function getLocalizationClassNameFromStatus<TLocalization extends EntityLocalization>(
+    language: LocalizationLanguage,
+    localizedEntity: EntityWithLocalizations<TLocalization>,
+) {
+    const entityLocalization = localizedEntity.localizations.find(
+        (localization) => localization.language.code === language.code,
+    );
+
+    switch (entityLocalization?.translationStatus) {
+        case TranslationStatus.Relevant:
+            return 'status-relevant';
+
+        case TranslationStatus.Outdated:
+            return 'status-outdated';
+
+        default:
+            return 'status-missing';
+    }
+}
```

Comment:
I believe it can be simplify, please double check it
Also helper functions should be above component
```suggestion
const getTranslationStatus = <TLocalization extends EntityLocalization>(
    languageCode: string,
    localizations: TLocalization[]
): TranslationStatus | undefined => {
    return localizations.find(
        (loc) => loc.language.code === languageCode
    )?.translationStatus;
};
```

Reply:
I check your suggestion and i work but I think my solution more easy to uderstand. So I want to kepp my solution but give me feedback about it.